plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/messages.properties

```diff
@@ -1,12 +1,14 @@
+BUTTON_ACCOUNTS_PANEL_ADD_ACCOUNT=Add Account
+BUTTON_ACCOUNTS_PANEL_LOGOUT=Sign Out
 LOGIN_ERROR_CANNOT_OPEN_BROWSER=Error opening a system browser. Signing in requires an open browser.
 LOGIN_ERROR_DIALOG_MESSAGE=An error occurred while trying to sign in.
 LOGIN_ERROR_DIALOG_TITLE=Sign in Failed
 LOGIN_ERROR_LOCAL_SERVER_RUN=Error listening for authorization code:\n\n
-LOGIN_MENU_LOGGED_IN=Sign out from Google
+LOGIN_MENU_LOGGED_IN=Sign in to/Sign out from Google...
```

Comment:
I don't understand this message. It feels like it should be one or the other. If the user can do both, this should be two separate menu items.